gub/config_cache.py

```diff
@@ -207,11 +207,14 @@
 
 config_cache['linux-64'] = config_cache['linux-x86']
 config_cache['freebsd-64'] = config_cache['freebsd-x86']
+config_cache['darwin-64'] = config_cache['darwin-x86']
 
 config_cache['arm'] += config_cache['32']
 config_cache['cygwin'] += config_cache['32']
 config_cache['darwin-ppc'] += config_cache['32']
+config_cache['darwin-64'] = config_cache['darwin-x86'] + config_cache['64']
 config_cache['darwin-x86'] += config_cache['32']
+config_cache['darwin-64'] += config_cache['64']
```

Comment:
Is this right?  AFAICS, `config_cache['64']` is now added twice...

Reply:
Good catch. I think one came from me and one from @Jahrme when I merged his code into my branch.

Reply:
Good catch. I think one came from me and one from @Jahrme when I merged his code into my branch.

Reply:
The duplication shouldn’t actually cause any problems, but I’ll get rid of it at some point.

Reply:
`config_cache['darwin-64'] = config_cache['darwin-x86']` is also assigned above, so actually line 215 can go away AFAICT

Reply:
@hahnjo Exactly.